### PR TITLE
AN-405 Added metrics

### PR DESCRIPTION
**Pull request by peterpf**
Issue: https://bitmovin.atlassian.net/browse/AN-405
Work:
  - Added metrics. In order to use them, select `Metrics` as aggregation and then one of the available metrics as dimension.
  - Generate the aggregation select-field from an array

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -96,11 +96,50 @@ function getDimensions() {
...
+ * Returns the available metrics the user can use to query API results for the metrics dimension.
+ *
+ * @returns {Array} Array of the available metrics.
+ */
+function getMetrics() {
+  if (this.metrics == null) {
+    this.metrics = [
+      {label: "Max Concurrent Viewers", value: "max_concurrentviewers"},
+      {label: "Avg Concurrent Viewers", value: "avg_concurrentviewers"},
+      {label: "Avg Dropped Frames",     value: "avg_dropped_frames"}
```
Comment (Tigraine) [inline src/main.js]: Let's remove the dropped frames for now

Comment (peterpf) [reply src/main.js]: done

**Comment by rolandgriesser** [comment]
If we would just have the metrics as upper case at the end of the list, would that be enough? we also don't pretty-print the normal dimensions. That would make the code less complicated, as you would only have to look up in a normal array if it's a metric or a normal dimension. But looks good to me otherwise.

@Tigraine your opinion?

**Review by sabe25 — APPROVED** [review]
LGTM

**Review by rolandgriesser — APPROVED** [review]

**Comment by rolandgriesser** [comment]
I've updated the code so it's a bit easier to read, maybe you can take another look @peterpf and then merge and release it? :)

**Pull request merged**